src/Authentication/Authenticators/Session.php

```diff
@@ -495,9 +495,12 @@ private function checkRememberMe(): bool
 
     private function getRememberMeToken(): ?string
     {
-        helper('cookie');
+        /** @var IncomingRequest $request */
+        $request = service('request');
 
-        return get_cookie('remember');
+        $cookieName = setting('Cookie.prefix') . setting('Auth.sessionConfig')['rememberCookieName'];
```

Comment:
This might be a good candidate for a Config method, like we did with redirect URL. At this point it looks like this is the only place it is used.